### PR TITLE
Add "copy to clipboard" button to the script console output

### PR DESCRIPTION
**Pull request by NotMyFault**
The script console output does now have a "copy to clipboard" button to grab the output with ease, when following the [issue reporting](https://www.jenkins.io/participate/report-issue/#Howtoreportanissue-WhatinformationtoprovideforEnvironmentandDescription) guide for example.:

### Proposed changelog entries

* Add a "copy to clipboard" button to the script console output.

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [X] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change) and are in the imperative mood. [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade

### Desired reviewers

@jenkinsci/sig-ux

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are accurate, human-readable, and in the imperative mood
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

File: core/src/main/resources/lib/hudson/scriptConsole.jelly
Comment (timja) [inline core/src/main/resources/lib/hudson/scriptConsole.jelly:58]: can you use css rather than a nbsp space?

Comment (NotMyFault) [reply core/src/main/resources/lib/hudson/scriptConsole.jelly:58]: The nbsp is actually obsolete, there's already a space introduced.

**Review by timja — APPROVED** [review]

**Comment by mPokornyETM** [comment]
Good idea

**Review by res0nance — APPROVED** [review]

**Comment by timja** [comment]
This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Pull request merged**
